modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -82,11 +82,9 @@ podman push --remove-signatures registry.redhat.io/rhel9/postgresql-15:latest de
 +
 For more information about pushing images directly to the {ocp-short} image registry, see https://access.redhat.com/solutions/6959306[How do I push an Image directly into the OpenShift 4 registry].
 +
-[TIP]
+[IMPORTANT]
 ====
-If an x509 error occurs, ensure you install the CA certificate used for {ocp-short} routes on your system.
-
-Optionally, you can append `--tls-verify=false` to the `podman` push command, note that this approach is not recommended.
+If an x509 error occurs, verify that you have installed the CA certificate used for {ocp-short} routes on your system.
```

Comment:
Not strictly in the scope of this PR, but I was wondering if we could probably add a link to a RH doc explaining how to install this CA certificate.
WDYT? Found this doc: https://access.redhat.com/solutions/6088891

Reply:
I think that would be a good idea actually - unfortunately I am having some issues accessing the customer portal today, but I will take a look at the article and OCP docs and see if there are some good links we can include 🙂